From a23b32e26f94b035c5c9e7075d6b5f738b109630 Mon Sep 17 00:00:00 2001
From: Sergii Pylypenko <x.pelya.x@gmail.com>
Date: Sun, 2 May 2021 23:30:34 +0300
Subject: [PATCH 0652/1053] Revert "Emscripten: fullscreen by default" - SDL2
 fullscreen flag is emulated, it won't make browser window borderless

This reverts commit abbe0a4a0afbc41ddcb167e43cb6775fc20b9308.
---
 src/table/misc_settings.ini | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/table/misc_settings.ini b/src/table/misc_settings.ini
index d4edd41d37..935993c7c2 100644
--- a/src/table/misc_settings.ini
+++ b/src/table/misc_settings.ini
@@ -60,7 +60,7 @@ full     = ""SHOW_TOWN_NAMES|SHOW_STATION_NAMES|SHOW_SIGNS|FULL_ANIMATION||FULL_
 [SDTG_BOOL]
 name     = ""fullscreen""
 var      = _fullscreen
-def      = true
+def      = false
 cat      = SC_BASIC
 
 [SDTG_BOOL]
-- 
2.55.0

